### ISSUE TITLE
Rendering a video effect flickers and stuck in 2019.61

### ISSUE THREAD

**Issue by mikedahm** [smeighan/xLights#1795]
I have a sequence from Magic Light Show (I can't upload their sequence as it is a purchase).  Rendering videos on my matrix worked prior to 2019.61.  This also happens opening a new blank sequence and putting a Video effect on the matrix with nothing else in the sequence.  No .mp4 or .mov files will play.  They flash and flicker but do not move or play.  Rendering also does not fix this issue from the preview.  Some only show a small amount at the top or bottom, others show a freeze of the first frame that flickers and will not move.  

I am using the 64bit Windows version of 2019.61, it does not crash when it shows this behavior.  Rolling back to 2019.60 renders videos in that sequence and in a blank sequence properly.
![Video2](https://user-images.githubusercontent.com/3976794/67828133-49b7bb80-faa0-11e9-8137-e8d55c54e4fb.png)
![Video1](https://user-images.githubusercontent.com/3976794/67828134-49b7bb80-faa0-11e9-8022-44f1c8185da7.png)

![Video3](https://user-images.githubusercontent.com/3976794/67828173-7370e280-faa0-11e9-8299-d0a54156529d.png)

**Comment by AzGilrock** [comment]
I guess you've been asleep and not reading anything that's been posted?  There was a new video hardware accelerator turned on that is causing issues.  You can disable it under Settings.  I believe next release will turn it off by default instead of on.

**Comment by mikedahm** [comment]
I did not see any posts about this issue from before posting this bug report.  I did see some people mention that their computer rendered slower to turn off hardware acceleration.  I tried looking through the xlights support facebook page but there are so many off topic posts that searching for anything with "video" just brought up a bunch of people showing off their shows.  I see newer posts from today mentioning the hardware accelerator and turning it off.  Thank you for the quick reply.
